lib/spack/spack/package.py

```diff
@@ -1814,7 +1814,7 @@ def build_log_path(self):
         if self.installed:
             return spack.store.layout.build_log_path(self.spec)
         else:
-            return os.path.join(self.stage.path, 'spack-build.out')
+            return os.path.join(self.stage.path, 'spack-build.txt')
```

Comment:
Why not return self.log_path() here?  That might take care of the code coverage issue.

Reply:
Thanks, looks like that did the trick!